mission_control/src/mission_control.cpp

```diff
@@ -428,10 +279,14 @@ int main(int argc, char** argv)
   ROS_INFO("Starting Mission mgr node Version: [%s]", NODE_VERSION);
   nh.setParam("/version_numbers/mission_control_node", NODE_VERSION);
 
-  MissionControlNode mmn(nh);
-  ros::spin();
-
-  ROS_INFO("Mission mgr shutting down");
+  MissionControlNode mcn(nh);
+  ros::Rate loop_rate(10);
+  while (ros::ok())
+  {
+    ros::spinOnce();
+    loop_rate.sleep();
+  }
```

Comment:
@patapufate why do we have a `main` here?

Reply:
Forgot to remove it